projects/grafana/grafana.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: DeploymentConfig
```

Comment:
You can collapse this in the manifest above (`grafana-ocp.yaml`) and remove the replication controller (since a deployment config manages replication controllers).